comfy_cli/command/generate/upload.py

```
@@ -0,0 +1,124 @@
+        _put_bytes(upload_url, data, ctype)
+    download_url = signed.get("download_url")
+    if not download_url:
+        raise client.ApiError(0, str(signed), "Server response missing download_url")
+    return UploadResult(
+        url=download_url,
+        expires_at=signed.get("expires_at"),
+        existing_file=bool(signed.get("existing_file", False)),
+    )
+
+
+def upload_path(path: Path | str, api_key: str) -> UploadResult:
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)